runtime/autoload/remote/define.vim

```diff
@@ -243,6 +243,10 @@ function! s:GetAutocmdPrefix(name, opts)
     call add(rv, 'nested')
   endif
 
+  if has_key(a:opts, 'once') && a:opts.once
+    call add(rv, 'once')
```

Comment:
Should this be `++once`? IIRC Vim supports `nested` (without the `++`) for backward compatibility reasons, but the same is not true for `once`.

While we're at it can you change `nested` on line 243 to `++nested` for consistency?

Reply:
@gpanders I see and understand background context. Will change

Reply:
Just in case,

> While we're at it can you change nested on line 243 to ++nested for consistency?

You means `++once`?